Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -540,44 +292,411 @@ def addSeeding(
         s.addAlgorithm(parEstimateAlg)
 
         if outputDirRoot is not None:
-            outputDirRoot = Path(outputDirRoot)
-            if not outputDirRoot.exists():
-                outputDirRoot.mkdir()
-            s.addWriter(
-                acts.examples.TrackFinderPerformanceWriter(
-                    level=customLogLevel(),
-                    inputProtoTracks=inputProtoTracks,
-                    inputParticles=selectedParticles,  # the original selected particles after digitization
-                    inputMeasurementParticlesMap="measurement_particles_map",
-                    filePath=str(outputDirRoot / "performance_seeding_trees.root"),
-                )
+            addSeedPerformanceWriters(
+                s,
+                outputDirRoot,
+                inputProtoTracks,
+                selectedParticles,
+                inputParticles,
+                parEstimateAlg.config.outputTrackParameters,
+                parEstimateAlg.config.outputProtoTracks,
+                logLevel,
             )
+    return s
 
-            s.addWriter(
-                acts.examples.SeedingPerformanceWriter(
-                    level=customLogLevel(minLevel=acts.logging.DEBUG),
```

Comment:
It was deliberate actually. It looked to me like a leftover from some debugging. Does it produce useful messages in regular running?

Reply:
@adye - just tell me if this is useful to have back for some reason.

Reply:
I don't know. I think this was there from the beginning in `seeding.py` and survived all the refactorings since.

It could make a difference since `addSeeding` in `seeding.py` is called with `acts.logging.VERBOSE`. Maybe the `minLevel` setting prevents too much detail from the `SeedingPerformanceWriter` part, but I haven't tried that.

Reply:
ok, let's restore it